cmd/lotus-seal-worker/main.go

```diff
@@ -48,19 +48,19 @@ func main() {
 	}
 
 	app := &cli.App{
-		Name:    "lotus-seal-worker",
+		Name:    "lotus-worker",
 		Usage:   "Remote storage miner worker",
 		Version: build.UserVersion(),
 		Flags: []cli.Flag{
 			&cli.StringFlag{
-				Name:    FlagStorageRepo,
-				EnvVars: []string{"WORKER_PATH"},
+				Name:    FlagWorkerRepo,
+				EnvVars: []string{"LOTUS_WORKER_PATH"},
```

Comment:
We can support multiple environment variables and add the old flag to the `Aliases` to allow for a deprecation period. We can then use the urfave.cli [BeforeFunc](https://godoc.org/github.com/urfave/cli#BeforeFunc) to add a deprecation warning for each environment variable.

We should strive to ensure that nothing breaks outside of the binary names for a short period.